terraform/main.tf

```diff
@@ -204,6 +204,21 @@ data "aws_iam_policy_document" "read_datadog_api_key_secret" {
   }
 }
 
+resource "aws_ses_receipt_rule" "ffis_ingest" {
+  name          = "ffis_ingest-${var.environment}"
+  rule_set_name = "ffis_ingest-rule-set"
+  recipients    = [var.ffis_ingest_email_address]
+  enabled       = true
+  scan_enabled  = true
+  tls_policy    = "Require"
+
+  s3_action {
+    bucket_name       = module.grants_source_data_bucket.bucket_id
+    position          = 1
+    object_key_prefix = "/ses/ffis_ingest/new"
```

Comment:
As we discussed in Slack, there's not a great way to test this outside of staging at the moment. However, dropping a note here just to keep it top-of-mind as something to check if we see issues – I've run into a few inconsistencies lately where S3 does (not) like the leading `/` at the beginning of object keys.

Reply:
Yeah, good catch. Let me change that before we roll this out.